quesma/telemetry/phone_home.go

```diff
@@ -348,10 +348,10 @@ func (a *agent) CollectClickHouse(ctx context.Context) (stats diag.ClickHouseSta
 	// https://gist.github.com/sanchezzzhak/511fd140e8809857f8f1d84ddb937015
 	stats.Status = statusNotOk
 
-	dbStats := a.clickHouseDb.Stats()
+	//dbStats := a.clickHouseDb.Stats(). TODO: after moving to generic impl (quesma_api.BackendConnector) we lost this driver-specific call
```

Comment:
I think, we should extend our API with `Stats` call

Reply:
++, though not sure yet about more general implementation - e.g. would it be fine to return just `map[string]any` or sth else

Reply:
hmm, Maybe we could return something similar do `DBStats` https://pkg.go.dev/database/sql#DB.Stats

Reply:
https://github.com/QuesmaOrg/quesma/pull/1163/commits/64927185b408374183c6e765d0b0945692c3cfe3